packages/ui5-application-writer/templates/core/README.md

```diff
@@ -0,0 +1,40 @@
+## Application Details
+|               |
+| ------------- |
+|**Generation Date and Time**<br><%= genDate %>|
```

Comment:
Not sure that the template should contain any references to anything other than UI5 application properties since this is the ui5-application-writer module. Here I see references to the generator, for example. These would be added by the generator...I think this should be far more generic.

Reply:
Moved to fiori generator shared as the read me  template makes use of generator-specific variables